scripts/circle-ci-tests.sh

```diff
@@ -36,7 +36,7 @@ if [ "$CIRCLE_NODE_TOTAL" == "1" ] ; then
     echo "To run in more containers, configure parallelism for this repo's settings "
     echo "via the CircleCI UI and adjust scripts/circle-ci-tests.sh to match."
 
-    echo "Running tests for common/lib/ and pavelib/"
+    echo "Running tests for pavelib/"
```

Comment:
can you update the comment on line #85 of this file, which says ` # run the commonlib unit tests`?

Reply:
Updated.